libsplinter/src/admin/service/messages/mod.rs

```diff
@@ -85,6 +86,17 @@ impl CreateCircuit {
             }
         };
 
+        let circuit_status = match proto.get_circuit_status() {
+            admin::Circuit_CircuitStatus::ACTIVE => CircuitStatus::Active,
+            admin::Circuit_CircuitStatus::DISBANDED => CircuitStatus::Disbanded,
+            admin::Circuit_CircuitStatus::ABANDONED => CircuitStatus::Abandoned,
+            admin::Circuit_CircuitStatus::UNSET_CIRCUIT_STATUS => {
```

Comment:
I think this needs to default to Active for 0.4 circuit compatibility. Since those circuits will not have this set.

Reply:
I changed this to default to active if not set